contracts/claim/Cargo.toml

```diff
@@ -40,17 +40,17 @@ optimize = """docker run --rm -v "$(pwd)":/code \
 """
 
 [dependencies]
-cosmwasm-std = { version = "1.0.0-beta7" }
-cosmwasm-storage = { version = "1.0.0-beta7" }
-cw2 = "0.13.1"
-cw-utils = "0.13.1"
+cosmwasm-std = { version = "1.0.0-beta8" }
+cosmwasm-storage = { version = "1.0.0-beta8" }
+cw2 = "0.13.2"
+cw-utils = "0.13.2"
 schemars = "0.8"
 serde = { version = "1.0", default-features = false, features = ["derive"] }
 minter = { path = "../minter", features = ["library"] }
-sg-std = { path = "../../packages/sg-std" }
+sg-std = "0.12.0"
```

Comment:
whats the difference? do we to stick to latest version or to a specific version?